backend/src/apiserver/server/experiment_server_test.go

```diff
@@ -341,21 +341,21 @@ func TestCreateExperimentV1_Multiuser(t *testing.T) {
 			"",
 		},
 		{
-			"Invalid - wrong owner",
+			"Invalid - wrong owner type",
 			DefaultFakeIdEight,
 			&apiv1beta1.Experiment{
 				Name:        "exp6",
 				Description: "first experiment",
 				ResourceReferences: []*apiv1beta1.ResourceReference{
 					{
-						Key:          &apiv1beta1.ResourceKey{Type: apiv1beta1.ResourceType_EXPERIMENT, Id: "exp2"},
+						Key:          &apiv1beta1.ResourceKey{Type: apiv1beta1.ResourceType_EXPERIMENT, Id: "ns1"},
```

Comment:
This diff looks fishy and possibly an indication of bugs somewhere. `ns1` is we used for namespace value, while this resource is of type experiment.

Reply:
It was intended to test a wrong resource type (`ResourceType_EXPERIMENT` instead of `ResourceType_NAMESPACE`), hence I changed it to `ns1`. Let me revert this change.